### ISSUE TITLE
Bomb Planted

### ISSUE THREAD

**Issue by CasaRica** [B3none/cs2-retakes#37]
At the beginning of each round say for example "[Retakes] Bomb Planted for: Short" but only TRs see the message.

**Comment by zwolof** [comment]
This isn’t too hard to implement, the issue is translating every single one, on every single map, for every single language.

**Comment by zwolof** [comment]
> > This isn’t too hard to implement, the issue is translating every single one, on every single map, for every single language.
> 
> I know it takes a little work to do for each map, but in terms of translations just only in EN and the community helped to do the rest ofd the translations for PT, RUS,....

Well yeah I can give it a shot, but it’s there’s gonna be a lot to translate. Might just implement it and let people make PRs if they want it to be available in their language

**Comment by B3none** [comment]
Yeah, this is a pretty tedious thing to add which is why I've not done it straight away for you
